dev_apps/acl-daemon/style/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  background-color: black;
+  font-family: "Lucida Console", Monaco, monospace;
+  color: green;
+}
+
+button {
+  font-size: 30px;
```

Comment:
nit: I know it's just a dev app, but I'd still prefer to just use rems because we're supposed to use them everywhere.